src/main/java/com/glencoesoftware/ldaptool/Main.java

```diff
@@ -20,101 +20,84 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.ldap.core.DistinguishedName;
 import org.springframework.ldap.core.LdapTemplate;
 
 import ch.qos.logback.classic.Level;
 import ome.logic.LdapImpl;
-import ome.logic.LdapImpl.GroupLoader;
-import ome.model.meta.Experimenter;
 import ome.system.OmeroContext;
 
 import java.io.File;
 import java.io.FileInputStream;
-import java.lang.reflect.Constructor;
-import java.lang.reflect.Field;
-import java.util.Arrays;
-import java.util.List;
 import java.util.Properties;
-import java.util.StringJoiner;
-import java.util.concurrent.Callable;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 import picocli.CommandLine;
-import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Command;
 import picocli.CommandLine.Option;
-import picocli.CommandLine.Parameters;
+import picocli.CommandLine.ParseResult;
 
 /**
  * @author Chris Allan <callan@glencoesoftware.com>
  */
-public class Main implements Callable<Integer>
+@Command(
+    mixinStandardHelpOptions = true,
```

Comment:
:+1: to adding this command attribute to include the standard help/version arguments.

We might need to implement `IVersionProvider` to read the version from the manifest as `-V` is returning nothing at the moment - https://picocli.info/#_dynamic_version_information